### PR TITLE
Add past quests

### PR DESCRIPTION
**Pull request by META-DREAMER**
This adds back in almost all of the past initiatives / quests, making use of inline-contributions for things like the Gitcoin Grants round4 donations.

However, the inline contributions are a bit buggy so the proper XP amounts aren't showing on the main page. This is a Draft PR until this issue is resolved.

Issues / things that are still missing:
- Some missing people / accounts from Intercon
- Some more recently started quests
- Inline contributions not working properly for Github usernames
- Quest1 completions
- prolly lots other stuff, let me know here

### CODE REVIEW

**Comment by yeqbfgxjiq** [comment]
@hammadj Do you think the bug will be fixed shortly, or should we merge this in so that we can create a more comprehensive Initiatives overview and tutorial sooner than later?

File: initiatives/2020-01-sc-partnership.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "weight": {
+      "incomplete": 13,
+      "complete": 13
+    },
+    "_fibonacciWeight": 13,
+    "completed": true,
+    "champions": [
+      "https://forum.metagame.wtf/u/decentralion",
+      "https://forum.metagame.wtf/u/burrrata"
+    ],
```
Comment (teamdandelion) [inline initiatives/2020-01-sc-partnership.json]: IMO @hammadj was a big champion here.

**Review by teamdandelion — COMMENTED** [review]
What bugginess are you seeing with inline contributions? cc @Beanow

**Comment by META-DREAMER** [comment]
> What bugginess are you seeing with inline contributions? cc @Beanow

The main issue is with the gitcoin donations initiatives. For example, oovg and pet3rpan's donations earned a lot of XP:
image

But that XP doesnt show up in the main list of users: 
image


Not sure if the weights are setup wrong, or if its not able to correctly map the contributor to the identity. Is there such thing as champions for inline initiatives? or is the contributors the only way to specify who did what inline initiative?

**Pull request merged**